packages/paste-style-props/src/types/background.ts

```diff
@@ -9,11 +9,11 @@ export type BackgroundColorOptions = keyof ThemeShape['backgroundColors'];
 export type BackgroundColor = ResponsiveValue<BackgroundColorOptions | 'none' | 'transparent' | 'inherit'>;
 
 // CSS native
-export type BackgroundImageOptions = CSS.BackgroundImageProperty;
-export type BackgroundSizeOptions = CSS.BackgroundSizeProperty<TLengthStyledSystem>;
-export type BackgroundPositionOptions = CSS.BackgroundPositionProperty<TLengthStyledSystem>;
-export type BackgroundRepeatOptions = CSS.BackgroundRepeatProperty;
-export type BackgroundAttachmentOptions = CSS.BackgroundAttachmentProperty;
+export type BackgroundImageOptions = Properties<TLengthStyledSystem>['backgroundImage'];
```

Comment:
Question: Is the reuse of the `<TLengthStyledSystem>` type argument / annotation on things that are not length or width type properties intentional? It was only ever used on widths, heights and other box like length dimensions to cast them to styled system.

Reply:
No, this was totally me making a best guess 😅 I wanted to ask about that in follow up. I'm guessing they are now removed, but if not, go forth.